doc/user/isisd.rst

```diff
@@ -261,10 +261,11 @@ ISIS interface
 
    Limit Remote LFA PQ node selection within the specified metric.
 
-.. clicmd:: isis fast-reroute ti-lfa [level-1|level-2] [node-protection]
+.. clicmd:: isis fast-reroute ti-lfa [level-1|level-2] [node-protection] [link-fallback]
```

Comment:
nit: `[node-protection [link-fallback]]` as in the command definition.

Reply:
Thanks! fixed.